lang/en.json

```diff
@@ -759,6 +759,7 @@
         "TagsLastUpdatedDescription": "The timestamp of the tag’s last modified date.",
         "TagsNameDescription": "The unique user-defined tag label for identifying and differentiating tags.",
         "TagNameHelp": "The tag name lets you define a unique name for this tag to quickly identify it. Please note that this name will be publicly visible in the source code of the tag when you preview/debug this container.",
+        "TagNameHelpNew": "Define a unique tag name. Note: This is publicly visible when the container is in preview/debug mode.",
```

Comment:
Tiny thought for the future (no changes needed now!): What might this key become after another rename? TagNameHelpNewNew? TagNameHelpNewer? TagNameHelpNew2? For sustainability, something like TagNameHelp_v2 or TagNameHelpV2 could work well. But all good for this PR!

Reply:
@araichyk Will change it to `TagManager_TagNameHelpV2` and good to use the same for future references